install/0000_80_machine-config_03_rbac.yaml

```diff
@@ -71,3 +71,89 @@ subjects:
 - kind: ServiceAccount
   name: prometheus-k8s
   namespace: openshift-monitoring
+---
+# Role host-networking-services lets system:node read config maps. This is needed in order to allow
+# configuring log level (and in the future more parameters) of static pods deployed in the
+# openshift-*-infra namespace.
+# Because host networking components right now only run on on-prem platforms, we create RoleBinding
+# explicitly only in namespaces for OpenStack, BareMetal, vSphere and Nutanix.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
why is this a clusterRole and not a role? This increases risk of sensitive config being exposed if a single node is compromised?

Reply:
I'd expect system:nodes to enable this already but only for CMs on pods bounded to the particular Node?

Reply:
>I'd expect system:nodes to enable this already but only for CMs on pods bounded to the particular Node?

As you wrote, system:nodes has it but only if there is CM on a pod bound on the node. But we run static pods, so at the time when they are created there is no notion of mounting any config map (i.e. we can't mount CM to a static pod in k8s world)

Reply:
>why is this a clusterRole and not a role? This increases risk of sensitive config being exposed if a single node is compromised?

I did it only for simplicity so that I create 1 role and N bindings in respective namespaces instead of N roles and N bindings (one pair per namespace).

Given that stuff in those `*-infra` namespaces runs as privileged workload with basically full access, I do not think this particular ClusterRole is anyhow increasing the surface for attack. But I am anyway okay with refactoring this to have separate Role per namespace and RoleBinding.